### DISCUSSION TITLE
"0px": "0px" in tailwind.config

### DISCUSSION THREAD

**Discussion by claudia-meneghesso** [tailwindlabs/tailwindcss#4586] [Help] [answered]
I started declaring pixel-perfect measurements in my tailwind.config file in the theme/spacing section. 

Strange enough, I since then was not able to use values such as `p-0` unless I would specifically declare it in the same section ("0px": "0px"). Is this supposed to be this way?

**Comment by VicGUTT** [comment, marked as answer]
Hey !

How are you defining the spacing values ?

Is it like this:
```js
theme: {
    // This will override all default Tailwind spacing values
    spacing: {
        // ...
    },
},
```
or like this:
```js
theme: {
    extend: {
        // This will "extend" (add to) all default Tailwind spacing values
        spacing: {
            // ...
        },
    },
},
```

Some useful details on the differences between the two:
- https://youtu.be/0l0Gx8gWPHk?t=377
- https://tailwindcss.com/docs/customizing-spacing#extending-the-default-spacing-scale
- https://tailwindcss.com/docs/customizing-spacing#overriding-the-default-spacing-scale

Hope it helps :)

**Reply by claudia-meneghesso** [reply]
Hi Vic!

Thanks again. Indeed, I was not extending them, I was overwriting them :)

**Reply by VicGUTT** [reply]
Hey ! 

Ahaha cool, it happens. Good thing Tailwind authors thought about it and gave us both options :)

If it's all good for you, would you mind marking the discussion as "answered" ? :)

**Reply by claudia-meneghesso** [reply]
Sure, thanks again!
